pages/doc/wavefront_release_notes.md

```diff
@@ -26,21 +26,55 @@ For information about the two subscription types and how they differ, see [Subsc
+          You need the <b>Logs</b> permission to view the logs alerts.
+      </li>
+      <li>
+          You need the <b>Alerts</b> and <b>Logs</b> permissions to create and manage logs alerts.
+      </li>
+      <p>The steps to add roles and permissions differ for Onboarded and Original subscriptions. See add permissions details for <a href="csp_users_roles.html#create-edit-or-delete-a-custom-role">Onboarded subscriptions</a> and <a href="users_roles.html">Original subscriptions</a>.</p>
+  </ul>
+  {{site.data.alerts.end}}
+  ![A screenshot of the alert browser on the logs alerts tab.](images/logs_alerts_for_rlease_notes.png)
+
 * **Maintenance Window Browser Page Improvements**: The [**Maintenance Window** browser page](maintenance_windows_managing.html#maintenance-windows) is now improved and allows you to:
 
-    * Hide and show details for all maintenance window or for a specific maintenance window.
-    * Sort the maintenance windows by name, state, start or end time, and so on.
-    * Apply various filters, hide and show the filters listed on the left.-->
+    * Hide and show details for all maintenance windows or for a specific maintenance window.
+    * Sort the maintenance windows by name, state, start or end time.
+    * Apply various filters, hide and show the filters listed on the left.
 
-*  **Google Cloud Platform Integration Improvements**: You can now select to ingest histogram metrics as well as filter these metrics by their Google Cloud Platform grouping function. See the [Integrations Release Notes](integrations_new_changed.html#july-2023) for details.
+* **New Field in the Generic Webhook Alert Template**: We added the [`contributingKVs` iterator](alert_target_customizing.html#list-all-sources-and-point-tags-of-an-aggregation-alerts-time-series), which returns the keys and values of each source and point tag used in the time series of a failed alert whose condition uses a single top-level aggregation function.
+
+  {% include important.html content="This feature is not available by default. To enable this feature for your service instance, contact your account representative or [technical support](wavefront_support_feedback.html#support)." %}
+
+* **Support for Alerting on ~alert.webhooks**: With this release, you can alert on `cs(~alert.webhooks.*)`. See [Query Responses of Webhook Alert Targets](webhooks_alert_notification.html#query-responses-of-webhook-alert-targets) for details.
+
+### Onboarded Subscriptions Only
 
-* **Documentation Improvement**: We have added a new [documentation page](integrations_gcp_api.html) which explains how to set up the Google Cloud Platform integration by using our REST API.
+**Support for Kubernetes Integration Setup**: We added support for setting up the Kubernetes integration when your Operations for Applications service is onboarded to VMware Cloud services. See the [Integrations ReleaseNotes](integrations_new_changed.html#august-2023) for details.
+
+## 2023-29.x Release Notes
```

Comment:
We need to include this information in 29.x RNs (below Original and Onboarded subscriptions only:

### Onboarded Subscriptions Only

* **Documentation Improvement**: We have added a new [documentation page](integrations_gcp_api.html) which explains how to set up the Google Cloud Platform integration by using our REST API.

I will put it back.